web/themes/joinup/prototype/sass/components/_ec-menu.sass

```diff
@@ -0,0 +1,33 @@
+// European Commision menu
```

Comment:
Not obligatory, but maybe you could add the footer in the list of components in the style guide similarly to the header.